src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4923,6 +4923,12 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="TypeMustBeVar" xml:space="preserve">
     <value>The type must be 'var'.</value>
   </data>
+  <data name="WRN_TupleLiteralNameMismatch" xml:space="preserve">
+    <value>The tuple literal should probably not use element name '{0}' at this position.</value>
+  </data>
+  <data name="WRN_TupleLiteralNameMismatch_Title" xml:space="preserve">
+    <value>The tuple literal should probably not use a mismatching element name at this position.</value>
```

Comment:
It is unused, but apparently there is a requirement that every `WRN_xyz` should also have a `WRN_xyz_Title`.
This is enforced by unittest.